doc-content/shared-kie-docs/src/main/asciidoc/KieServer/Install-section.adoc

```diff
@@ -84,197 +85,377 @@ container. The Web container version of the WAR contains only the REST interface
 . Verify the server is running. Go to `http://SERVER:PORT/CONTEXT/services/rest/server/` and type the specified username and password. You should see simple XML message with basic information about the server.
 +
 image::KieServer/kie-server-info.png[]
+endif::[]
 
+= {KIE_SERVER} system properties
 
-== Bootstrap switches
+The {KIE_SERVER} accepts a number of system properties (bootstrap switches) to configure the behavior of the server.
 
 
-The Kie Server accepts a number of bootstrap switches (system properties) to configure the behaviour of the server.
-The following is a table of all the supported switches.
+.System properties for disabling {KIE_SERVER} extensions
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.drools.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the Business Rule Management (BRM) support (for example rules support).
+
+ifdef::PAM,JBPM[]
+|`org.jbpm.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the {PRODUCT} support (for example processes support).
+
+|`org.jbpm.ui.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the {PRODUCT} UI extension.
+
+|`org.jbpm.case.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the {PRODUCT} case management extension.
+
+|`org.kie.executor.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the {KIE_SERVER} executor.
```

Comment:
This property should be moved to other executor properties, i.e. `org.kie.executor.interval`.

Reply:
Ack.